sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -1990,7 +1990,13 @@ def hpat_pandas_series_isin(self, values):
     def hpat_pandas_series_isin_impl(self, values):
         # TODO: replace with below line when Numba supports np.isin in nopython mode
         # return pandas.Series(np.isin(self._data, values))
-        return pandas.Series(data=[(x in values) for x in self._data], index=self._index, name=self._name)
+
+        data_len = len(self._data)
+        result = numpy.array([False] * data_len)
+        for i in numba.prange(data_len):
+            result[i] = self._data[i] in values
```

Comment:
Didn't you try to convert `values` to set?

Reply:
This slows down the execution of the function, even on lists with large repetitions.

Reply:
@Rubtsowa 
no, it doesn't

name|nthreads|type|size|median|min|max|compile|boxing
----|--------|----|----|------|---|---|-------|------
Series.isin|1.0|Python|10000000.0|0.009159|0.009127|0.009223|NaN|NaN
 | | 1.0 | SDC|10000000.0|0.027854|0.027554|0.028204|0.104338|0.000232
||8.0|Python|10000000.0|0.009403|0.008639|0.012322|NaN|NaN
||8.0|SDC|10000000.0|0.006398|0.006361|0.007950|0.106473|0.00027


Moreover, I would say, it speeds everything up

Reply:
Without set:

name|nthreads|type|size|median|min|max|compile|boxing
----|--------|----|----|------|---|---|-------|------
Series.isin|1.0|Python|10000000.0|0.008859|0.008839|0.008965|NaN|NaN
||1.0|SDC|10000000.0|0.127839|0.127763|0.127877|0.095031|0.000235
||8.0|Python|10000000.0|0.009093|0.008691|0.012178|NaN|NaN
||8.0|SDC|10000000.0|0.324509|0.322576|0.454564|0.08109|0.000293

Reply:
Initial version (list comprehension):

name|nthreads|type|size|median|min|max|compile|boxing
----|--------|----|----|------|---|---|-------|------
Series.isin|1.0|Python|10000000.0|0.005723|0.005515|0.006462|NaN|NaN
||1.0|SDC|10000000.0|0.252864|0.232145|0.253373|0.066061|0.000244
||8.0|Python|10000000.0|0.005694|0.005566|0.006526|NaN|NaN
||8.0|SDC|10000000.0|0.232615|0.232179|0.232622|0.06533|0.000257

Reply:
without set:
Ran 1 test in 0.505s

with set:
Ran 1 test in 1.676s

Reply:
Is it time of execution of unit test or perf. test? Why do you use `python.unittest` time from command line?